h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostNativeMojoModel.java

```diff
@@ -38,19 +39,24 @@ private static Booster makeBooster(byte[] boosterBytes) {
   }
 
   public final double[] score0(double[] doubles, double offset, double[] preds) {
-    return score0(doubles, offset, preds,
-            _boosterType, _ntrees,
-            _booster, _nums, _cats, _catOffsets, _useAllFactorLevels,
-            _nclasses, _priorClassDistrib, _defaultThreshold, _sparse);
+    return score0(
+        doubles, offset, preds,
+        _boosterType, _ntrees,
+        _booster, _nums, _cats, _catOffsets, _useAllFactorLevels,
+        _nclasses, _priorClassDistrib, _defaultThreshold, _sparse, 
+        _mojo_version
+    );
   }
 
-  public static double[] score0(double[] doubles, double offset, double[] preds,
-                                String _boosterType, int _ntrees,
-                                Booster _booster, int _nums, int _cats,
-                                int[] _catOffsets, boolean _useAllFactorLevels,
-                                int nclasses, double[] _priorClassDistrib,
-                                double _defaultThreshold, final boolean _sparse) {
-
+  public static double[] score0(
+      double[] doubles, double offset, double[] preds,
+      String _boosterType, int _ntrees, 
+      Booster _booster, int _nums, int _cats,
+      int[] _catOffsets, boolean _useAllFactorLevels,
+      int nclasses, double[] _priorClassDistrib,
+      double _defaultThreshold, final boolean _sparse, 
+      double _mojo_version
```

Comment:
can we avoid making it a parameter? for in H2O execution, we don't really need to add the version

you can move the check to the non-static method (line 42) and throw the message there

Reply:
👍 this was lazy of me

Reply:
fixed